client/src/pages/ExistingProgramme/editRequirementsBox.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
```

Comment:
Can we rename components files to be capitalised please :)

Reply:
done